src/wireviz/wv_helper.py

```diff
@@ -43,7 +45,8 @@ def nested_html_table(rows):
                 html.append('  <table border="0" cellspacing="0" cellpadding="3" cellborder="1"><tr>')
                 for cell in row:
                     if cell is not None:
-                        html.append(f'   <td balign="left">{cell}</td>')
+                        # Inject attributes to the preceeding <td> tag where needed
+                        html.append(f'   <td balign="left">{cell}</td>'.replace('><tdX', ''))
```

Comment:
the `<tdX` hack is clever, but I'm wondering if there is a more transparent way to show what's going on? I don't have a specific suggestion at this point, but will think about it.

Reply:
It's just a generalization of the `.replace()` from your code version. The selected tag name `<tdX>` is a bit cryptic and could be spelled out to e.g. `<join-with-previous-tag>`, but that would result in longer code lines that might harm readability. With all functions using this collected in a new module as suggested in #151, that would be a place to explain this implementation technique.

Reply:
Agreed. We can leave it as it is now, and clean up / improve / explain in #151 as you suggest.